backend/geneaprove/importers/gedcomimport.py

```diff
@@ -369,31 +360,30 @@ def _create_project(self, file, researcher):
         return p
 
     def unexpected(self, field):
-        self.report_error(field, "Unexpected tag %s" % field.tag)
+        self.report_error(field, f"Unexpected tag {field.tag}")
 
     def ignored(self, field, prefix):
         self.report_error(
             field,
-            "Ignored %s%s" % ('%s.' % prefix if prefix else '', field.tag))
+            f"Ignored {prefix + '.' if prefix else ''}{field.tag}")
```

Comment:
What's the different with:
    {prefix}{'.' if prefix else ''}{field.tag}
to avoid an 'ugly' + operator

Reply:
The primary difference is likely an insufficient coffee intake. `{prefix}{'.' if prefix else ''}` demonstrates much better performance (on the part of the programmer.) Changed.